my_list.rb

```diff
@@ -0,0 +1,31 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&number)
+    @list.each(&number)
```

Comment:
- [ ] I got an `undefined method `call' for nil:NilClass (NoMethodError)
` when I try to test your code. The error seems to come from line 11. Can you please check it out?

See image: 
![a12](https://user-images.githubusercontent.com/39523312/156645842-73a08aeb-14a6-4f96-af9e-c6dd893ad5b8.png)

Reply:
The error has been fixed.